files/ja/web/javascript/reference/global_objects/array/sort/index.md

```diff
@@ -52,27 +43,28 @@ sort(function compareFn(a, b) {
 
 `compareFn` が与えられなかった場合、`undefined` 以外のすべての配列要素は文字列に変換され、文字列が UTF-16 コード単位順でソートされます。例えば、"banana" は "cherry" の前に来ます。数値のソートでは、9 が 80 の前に来ますが、数値は文字列に変換されるため、Unicode 順で "80" が "9" の前に来ます。`undefined` の要素はすべて、配列の末尾に並べられます。
 
-> **メモ:** UTF-16 では、`\uFFFF` を超える Unicode 文字は 2 つのサロゲートコード単位にエンコードされ、`\uD800`-`\uDFFF` の範囲になります。それぞれのコード単位の値は比較では別々に扱われます。したがって、`\uD855\uDE51` というサロゲートペアで形成される文字は、`\uFF3A` の文字よりも前に並べられます。
+`sort()` メソッドは空のスロットを保持します。[疎配列](/ja/docs/Web/JavaScript/Guide/Indexed_collections#疎配列)の場合、空のスロットは配列の末尾に移動され、常にすべての `undefined` の後に置かれます。
+
+> **メモ:** UTF-16 では、`\uFFFF` を超える Unicode 文字は 2 つのサロゲートコード単位にエンコードされ、`\uD800` - `\uDFFF` の範囲になります。それぞれのコード単位の値は比較では別々に扱われます。したがって、`\uD855\uDE51` というサロゲートペアで形成される文字は、`\uFF3A` の文字よりも前に並べられます。
 
 `compareFn` が与えられた場合、`undefined` 以外のすべての配列要素は比較関数の返値に基づきソートされます（`undefined` の要素はすべて、`compareFn` を呼び出すことなく配列の末尾へ並べられます）。
 
 | `compareFn(a, b)` の返値 | ソート順                        |
 | ------------------------ | ------------------------------- |
-| > 0                      | `a` を `b` の後に並べる         |
-| < 0                      | `a` を `b` の前に並べる         |
+| > 0                      | `a` を `b` の後に並べる（例えば `[b, a]`） |
+| < 0                      | `a` を `b` の前に並べる（例えば `[a, b]`） |
 | === 0                    | `a` と `b` の元の順序を維持する |
```

Comment:
This is because the markdown table is not formatted. run `yarn prettier -w files/ja/web/javascript/reference/global_objects/array/sort/index.md` locally could resolve the lint error.

Reply:
The error has been resolved.
Thank you, @in1999 !